docs/overview_cjoyce.md

```diff
@@ -0,0 +1,36 @@
+# To-Do service API
+
+Organize your life with To-Do API! You can use this cloud-hosted to-do list to schedule your tasks and events, and receive helpful reminders at your chosen time.
+
+## Get started
+
+Learn how to do common tasks with the To-Do service using our tutorials!
+
+1. Use [this guide](before-you-start-a-tutorial.md) to set up your development system for these tutorials. You only have to do this once per development system.
+2. Use the tutorials in the drop-down below to complete your tasks!
+
+<details>
```

Comment:
You can't embed markdown in an HTML tag.
I converted the original markdown list to an HTML list.
Note that in an HTML list, you have to provide the correct _filename_ of the rendered file, which is different from what happens in markdown.